src/components/NavBar/SearchBar.tsx

```diff
@@ -102,7 +105,11 @@ export const SearchBar = () => {
     ...trace,
   }
   const placeholderText = useMemo(() => {
-    return isMobileOrTablet ? t`Search` : t`Search tokens and NFT collections`
+    return isMobileOrTablet
+      ? t`Search`
+      : shouldDisableNFTRoutes
+      ? t`Search tokens`
+      : t`Search tokens and NFT collections`
```

Comment:
These should not be translated - this is fixed separately in https://github.com/Uniswap/interface/pull/6720

Reply:
<img width="547" alt="Screenshot 2023-06-08 at 10 32 15 AM" src="https://github.com/Uniswap/interface/assets/35351983/85577c37-9fa6-4313-972d-78ee96695f2b">

Got it. Should I just ignore the second task then?

Reply:
You should update it, but you should first merge in main, where the translation issue is already fixed.